builder/vmware/common/driver_esx5.go

```diff
@@ -47,7 +47,7 @@ type ESX5Driver struct {
 
 func (d *ESX5Driver) Clone(dst, src string, linked bool) error {
 
-	linesToArray := func(lines string) []string { return strings.Split(strings.Trim(lines, "\n"), "\n") }
+	linesToArray := func(lines string) []string { return strings.Split(strings.Trim(lines, "\r\n"), "\n") }
```

Comment:
Quickly reading this, the old behaviour `strings.Trim(lines, "\n")` will not happen any more. Not sure if this could be an issue.

Reply:
nope, strings.Trim() goes character by character. Not sure why it doesn't ask for an array, but I didn't write it. 🤷‍♀️ 

https://play.golang.org/p/GRCAu4_fHVd